internal/prober/multihttp/script.tmpl

```diff
@@ -101,8 +101,10 @@ export default function() {
 		  name: '{{ $idx }}', // TODO(mem): give the user some control over this?
 		  __raw_url__: '{{ .Request.Url }}',
 		},
-		redirects: 0{{ if gt (len $headers) 0 }},
-		headers: {{ $headers }}{{ end }}
+		{{- if gt ($headers | len) 0 }}
```

Comment:
Uhm...

This is changing what this is doing.

The template is outputting:
```
redirects: 0
```

If `$headers` contains something, it does this:

```
redirects: 0,
headers: xxx
```

The new version is not outputting anything, except in the case where $headers contains something, in which case it outputs:

```
redirects: 0,
headers: xxx
```

The handling of the redirects needs fixing, but not like this.